python/pyspark/worker.py

```diff
@@ -44,7 +44,7 @@
 from pyspark.util import _get_argspec, fail_on_stopiteration
 from pyspark import shuffle
 
-if sys.version >= '3':
+if sys.version_info.major >= 3:
```

Comment:
Can we don't change this line?

Reply:
Ah, I forgot this line. Let me push a new commit.